sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActionResult.cs

```diff
@@ -13,9 +13,11 @@ public class TextAnalyticsActionResult
     {
         internal TextAnalyticsActionResult(string actionName, DateTimeOffset completedOn, TextAnalyticsErrorInternal error)
```

Comment:
I may be missing something, but if you removed `TextAnalyticsErrorInternal` then we can't have it in a method signature or things will not compile.    Can we check if this is still being referenced anywhere?   If not, it should be safe to delete.